hlxsites/merative2
# feat(back-to-top): remove back to top in favor of Drift

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-815

## Description

**Removed**

- This PR removes the `Back to Top` button functionality so that we can use the bottom right hand corner for the Drift Chatbot Widget.

## Design Specs

- Figma Link - N/A

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/
- After (Changes from this PR): https://feat-drift-remove-back-to-top--merative2--proeung.hlx.page/
  
## Testing Instruction
- Visit this deploy preview (https://feat-drift-remove-back-to-top--merative2--proeung.hlx.page).
- Ensure that the "Back to Top" button is no longer visible.
- Ensure that the Drift button is visible once, the playbooks configs have been implemented to the test URL above.
- https://feat-drift-remove-back-to-top--merative2--proeung.hlx.page/#bot_test-5086233-t9fx4CHq
